packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -351,3 +357,28 @@ if (bootstrapConfig) {
         Log.error('Sandbox initialization failed: ' + error.message);
     });
 }
+
+/**
+ * Handle higher layer changes when starting UI Adaptation.
+ * When RTA detects higher layer changes an error with Reload triggered text is thrown, the RTA instance is destroyed and the application is reloaded.
+ * For UI5 version lower then 1.84.0 RTA is showing a popup with notification text about the detection of higher layer changes.
```

Comment:
typo: `lower than`

Reply:
Fixed in [652a18e](https://github.com/SAP/open-ux-tools/pull/2536/commits/652a18e63fd64cf9f64e2a3b65d78254075b0fbb)